numba/cpython/setobj.py

```diff
@@ -1298,15 +1298,26 @@ def set_add(context, builder, sig, args):
 
     return context.get_dummy_value()
 
-@lower_builtin("set.discard", types.Set, types.Any)
+
 def set_discard(context, builder, sig, args):
```

Comment:
Move this into the intrinsic `_set_discard` as the `codegen` (and same comment for all the occurrences of this pattern)? i.e.
```python
@intrinsic
def _set_<something>(typingctx, <args>):
    sig = <>
    def codegen(cgctx, builder, sig, args): # This is the `set_<something>` function at present.
        ...
    return sig, codegen
```

Reply:
Ok! I kept this way to reduce the patch size and to make it easier to review.